src/main/java/com/traveltime/sdk/dto/requests/timefilterfast/Transportation.java

```diff
@@ -0,0 +1,21 @@
+package com.traveltime.sdk.dto.requests.timefilterfast;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.traveltime.sdk.dto.requests.timefilterfast.transportation.*;
+
+@JsonTypeInfo(
+    use = JsonTypeInfo.Id.NAME,
+    property = "type"
+)
+@JsonSubTypes({
```

Comment:
According to our API docs [1], time filter fast only supports `Possible values: public_transport, driving, driving+public_transport`.

I'm not sure which version is true though, @bartosz-witkowski ?

[1] https://docs.traveltime.com/api/reference/time-filter-fast#arrival_searches-many_to_one-transportation-type

Reply:
Right now we only have the four modes you mentioned but in the near future we'll be rolling out different modes.

Reply:
I mentioned three though :grimacing:

Reply:
Ermm derp. Also scratch what I said currently we support

-  `walking+ferry` (as a hack `walking` defaults to this)
- `cycling+ferry` (as a hack `cycling` defaults to this)
- `driving+ferry` (as a hack `driving` defaults to this)
- `driving+public_transport`
- `public_transport`

on the json api. For the proto API it should be the same.

Reply:
Ok, so this is fine, docs need to be updated.